lib/util/package-rules.ts

```diff
@@ -193,8 +193,9 @@ function matchesRule(
     positiveMatch = true;
   }
   if (matchSourceUrlPrefixes.length) {
+    const lowerCaseSourceUrl = sourceUrl.toLowerCase();
     const isMatch = matchSourceUrlPrefixes.some((prefix) =>
-      sourceUrl?.startsWith(prefix)
+      lowerCaseSourceUrl?.startsWith(prefix.toLowerCase())
```

Comment:
I think @viceice still wanted to have a test???

The PR title and PR body are good now.

Reply:
Yes, please add a unit test so we don't silently loose this in future